src/main/java/org/pesmypetcare/webservice/dao/usermanager/UserDaoImpl.java

```diff
@@ -51,6 +51,7 @@
     private static final String UPDATE_FAILED_CODE = "update-failed";
     private static final String FIELD_LIKED_BY = "likedBy";
     private static final String FCM = "FCM";
+    private static final String NOTIFICATIONS_FIELD = "notification-tokens";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String "notification-tokens" appears 2 times in the file.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5649852)